docs/pages/build/internal-distribution.mdx

```diff
@@ -3,168 +3,93 @@ title: Internal distribution
-    '$ eas build --profile preview --platform all',
-  ]}
-  cmdCopy="eas build --profile preview --platform all"
-/>
+To use Google Play beta, you need to upload your app as an AAB (Android App Bundle) to the Google Play Console, set up a test track, and invite users via email or a shareable link. Testers can install the app through the Play Store, and you can collect feedback and crash reports directly from the Google Play Console. If you intend to ship new builds frequently, investing in distributing APKs directly to testers will be worthwhile.
 
-> If you're using ad hoc provisioning but you haven't registered any devices yet, you'll be asked to register them now (or exit the current command and run `eas device:create` again). The build command will wait for the new device to register. Scan the QR code that is presented in the terminal and follow the instructions on that page to register your device. When you're done, return to the terminal and continue.
+</Collapsible>
 
-When the build completes, you will be given a URL that you can share with your team to download and install the app.
+<Collapsible summary="iOS: TestFlight">
 
-</Step>
+TestFlight is another option to distribute your app to iOS devices. TestFlight also requires a paid Apple Developer account, but allows you to share your app with up to 10,000 users that can be invited with their email or a public link. This method requires you to [upload your app](/submit/ios) to App Store Connect and wait for the automated review before you can share a build. If you intend to ship new builds frequently, investing the time to set up Ad Hoc distribution will be worthwhile.
 
-<Step label="4">
+</Collapsible>
 
-## Install and run the build
+## Internal distribution with EAS Build
 
-Press the "Install" button on the build page and follow the instructions presented in the modal.
+Setting up an internal distribution build only takes a few minutes with EAS Build and provides a streamlined way to share your app with your team and testers for feedback.
 
-If you're running iOS 16 or above and haven't yet turned on Developer Mode, you will need to [enable it](/guides/ios-developer-mode) before you can run your build. This doesn't apply if you're using enterprise provisioning.
+The setup requires configuring a build profile in the **eas.json** configuration file and setting `"distribution": "internal"` to a build profile. This configuration automatically generates **.apk** files for Android and **.ipa** files for iOS so that they can be installed on devices directly and do not require a development server to run the app.
 
-</Step>
+For Android, the setup uses the Android keystore or can re-use the previously generated keystore for a [development build](/develop/development-builds/introduction/). However, iOS requires an ad hoc provisioning profile that explicitly allows devices to run that app based on that device's UDID. This can be done using `eas device:create` when creating a new build.
```

Comment:
```suggestion
For Android, EAS Build will generate a new Android keystore for signing the APK, or it will use an existing one if the package name is the same as your [development build](/develop/development-builds/introduction/). For iOS, EAS Build will generate an ad hoc provisioning profile that explicitly allows devices to run that app based on that device's UDID. This can be done using `eas device:create` when creating a new build.
```